ui/component/publishFile/view.jsx

```diff
@@ -8,6 +8,7 @@ import Card from 'component/common/card';
 import { FormField } from 'component/common/form';
 import Spinner from 'component/spinner';
 import I18nMessage from '../i18nMessage';
+import usePersistedState from '../../effects/use-persisted-state';
```

Comment:
Can you change these to use absolute import paths?

Reply:
Fixed.